policy/modules/contrib/fdo.fc

```diff
@@ -1,10 +1,17 @@
+
+/etc/device-credentials				--	gen_context(system_u:object_r:fdo_conf_rw_t,s0)
```

Comment:
I also think we need to add `/boot/device-credentials` as that's where this file comes from

Reply:
thank you